### DISCUSSION TITLE
Autopilot STAR mess

### DISCUSSION THREAD

**Discussion by robinottawa** [flybywiresim/aircraft#3494] [Q&A] [answered]
(correct place to post question?)

I have not had an ILS landing work for me for at least a month - feels way longer. This last one is a good example. As I go past the last wp before the ILS, the AP disconnects and the next sp is shown as the airport/vor/?, the flight path becomes dotted instead of solid, and the aircraft begins to turn directly to the airport. Any suggestions or explanations would be appreciated. I'm hand flying every ILS these days. 

(I just noticed that the ILS runway indicated on the top of the nav display shows 15R, whereas the FP has 05. I entered 05 into the MCDU so maybe the databases are messed up?)

![5 2](https://user-images.githubusercontent.com/75288729/107826251-9afdc400-6d52-11eb-8a3d-9cb264b3015e.jpg)

**Comment by Biniou671** [comment]
I am having almost the same issue at the last star point, in Dev or experimental version, but for me the AP is still connected, then suddenly the plane turn left or right with bank angle above 30 degree, and going down for crashing. So I need to disconnect AP to stabilize the plane, then I need to amend flight plan in the MCDU with DIR option to the second waypoint of the approach itinerary.
Now, I came back to stable version 0.5.2. [missing the printer option... ;(   ]

**Comment by hotshotp** [comment, marked as answer]
Proper STARS are not implemented and will be in the future

**Reply by robinottawa** [reply]
This is a MSFS thing, I assume. How bizarre is it to have an aircraft that can't fly a route.

**Reply by hotshotp** [reply]
> This is a MSFS thing, I assume. How bizarre is it to have an aircraft that can't fly a route. 

Yep, msfs thing, and yes, very bizzare - but we are working on it! :D

**Comment by riveroalbert** [comment]
You can always fly non-RNAV STARs manually with selected heading, VORDME and the charts. Its a good way of practise before it gets implemented ;)

**Reply by robinottawa** [reply]
True. But it's pretty brutal that an airplane in a simulator cannot fly a route to land. It's MSFS, of course, and I appreciate the work FBW volunteers are doing.

**Comment by robinottawa** [comment]
I just tried it with the Stable version 5.2? and it's the same. Doesn't work for ILS.

**Reply by robinottawa** [reply]
Ok, so I went on discord and there is a list of "known issues" and it is on there. Posting the current list here in case people don't have access. I can't see how to link to it but there's something called a "message link" I'll post right here--->> https://discord.com/channels/738864299392630914/771435594445226005/771437342577131521

- AP not following the flight plan (leaking input values affect, but don't disconnect the AP) [Workaround: Set dead-zones for your input device higher] 

- Autopilot goes direct to RWY on APP (same with the default A320) [Workaround: Use DIR to a waypoint or selected heading]

- Upper ECAM displays wrong THR levers position / N1 rating.

- Newly added ASOBO A320 liveries are incompatible with the A32NX mod. 

- Distance remaining not updating 

 - Ruder keybindings not working  (you have to set your keybinding to rudder axis right and left)

- Wing dips on landing (due to bad transition to direct law in flare, same with the default A320)  [Workaround use minimal aileron input on landing]

- Black screens / unable to start  (conflict with another mod/livery or incorrect installation of the A32NX mod, use the installer linked at the bottom)
